pkg/pillar/cmd/volumemgr/handledownloader.go

```diff
@@ -10,15 +10,16 @@ import (
 	log "github.com/sirupsen/logrus"
 )
 
-func AddOrRefcountDownloaderConfig(ctx *volumemgrContext, status types.OldVolumeStatus) {
+// AddOrRefcountDownloaderConfig used to publish the downloader config
+func AddOrRefcountDownloaderConfig(ctx *volumemgrContext, status types.ContentTreeStatus) {
 
-	log.Infof("AddOrRefcountDownloaderConfig for %s", status.BlobSha256)
+	log.Infof("AddOrRefcountDownloaderConfig for %s", status.ContentSha256)
 
 	refCount := uint(1)
-	m := lookupDownloaderConfig(ctx, status.ObjType, status.BlobSha256)
+	m := lookupDownloaderConfig(ctx, status.ObjType, status.ContentSha256)
 	if m != nil {
 		log.Infof("downloader config exists for %s to refcount %d",
-			status.VolumeID, m.RefCount)
+			status.ContentID, m.RefCount)
```

Comment:
Shouldn't this log also be ablut the ContentSha256 like the others?

Reply:
Resolved.